### ISSUE TITLE
latitude and longitude fields

### ISSUE THREAD

**Issue by managingwholes** [Soil-Carbon-Coalition/atlasbiowork#16]
Most of our data will be point data, but it is great to be able to specify geometries for sites.

Can the latitude and longitude fields be made editable in boxed fields? What would this involve?

Also, on my Android phone the "number" input type does not give me a keypad with a negative sign, just 0-9 and a decimal point, so if I choose "Enter coordinates" on the locate template, I am stuck in the eastern hemisphere.

**Comment by sheppard** [comment]
Yes, this could be done by re-using or re-purposing the lat/long editor in "New Site + Observation".  One trick will be to convert seamlessly between a geometry field and separate lat/long fields.  I have one part of that conversion in the [code for saving](https://github.com/Soil-Carbon-Coalition/atlasbiowork/blob/master/db/obs/serializers.py#L53-L65) a new site + observation.  The other trick will be to have both Leaflet.draw and wq/locate running on the same map - it should be doable, but might take a little hacking.

For lat/long (and potentially all numbers) we might want to try using `<input type=tel>` rather than `<input type=number>`.  Using a "telephone" input often brings up a more usable keypad on mobile devices.
